apis/v1alpha2/config.go

```diff
@@ -128,7 +128,16 @@ func hasNullValue(cfg map[string]interface{}) []string {
 			nullKeys = append(nullKeys, fmt.Sprintf("%s:", k))
 		}
 		if reflect.ValueOf(v).Kind() == reflect.Map {
-			nulls := hasNullValue(v.(map[string]interface{}))
+			val := make(map[string]interface{})
+			switch t := v.(type) {
+			case map[string]interface{}:
+				val = t
+			case map[interface{}]interface{}:
+				for kk, vv := range t {
+					val[fmt.Sprintf("%v", kk)] = vv
```

Comment:
This looks suspicious. 

When does this happen? It would be good to have a test for this.

Reply:
Oh well, seems that its also no longer required. I replaced it with a default section that returns an empty array. Instead of panic if its not a `map[string]interface{}`